src/www/widgets/widgets/thermal_sensors.widget.php

```diff
@@ -130,7 +131,12 @@ function thermal_sensors_widget_update(sender, data) {
                     .addClass('progress-bar-success');
             }
             // update bar
-            progressBar.html(sensor['temperature'] + ' &deg;C');
+            
+            if($('#thermal_sensors_widget_show_temp_in_farenheit').attr('checked') === 'checked') {  
+                progressBar.html(((9/5) * sensor['temperature']+(32)) + ' &deg;F');
```

Comment:
You can use 1.8 to avoid the division. It is also possible to use the degree symbol as is as unnecessary HTML entities should be avoided. This comes from HTML4 and earlier times where Unicode was not that common.